src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/FinalLocalVariable/case1/context/defaultContext.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/Test.java b/Test.java
+index 2a731db..eb996ad 100644
+--- a/Test.java
++++ b/Test.java
+@@ -2,12 +2,12 @@ package TreeWalker.coding.FinalLocalVariable;
+ 
+ public class Test {
+     static int foo(int x, int y) {  // violation without filter
+-        x = 1;
+         return x+y;
+     }
+     public static void main (String []args) {  // violation without filter
+         for (String i : args) {
+             System.out.println(i);
++            System.out.println(i);
+         }
+         int result=foo(1,2);  // violation without filter
```

Comment:
>Test.java:12:13: Variable 'result' should be declared final.

~~why context strategy gives a violation on this line? Nothing changes here~~
ok, I understand now - the check was in neverSuppressedCheks